### PR TITLE
Allow empty access tokens

### PR DESCRIPTION
**Pull request by austinlparker**
This removes the check for access tokens in the `Validate` method and updates the example code.

### CODE REVIEW

File: examples/trivial.go [hunk 1]
```diff
@@ -45,18 +44,12 @@ func (r *LoggingRecorder) RecordSpan(span lightstep.RawSpan) {
...
-		os.Exit(1)
-	}
-
 	loggableRecorder := &LoggingRecorder{}
 
 	// Use LightStep as the global OpenTracing Tracer.
 	opentracing.InitGlobalTracer(lightstep.NewTracer(lightstep.Options{
 		AccessToken: *accessToken,
-		Collector:   lightstep.Endpoint{Host: "localhost", Port: 9997, Plaintext: true},
+		Collector:   lightstep.Endpoint{Host: "localhost", Port: 8360, Plaintext: true},
```
Comment (iredelmeier) [inline examples/trivial.go]: Can we separate this into a separate PR?

Comment (austinlparker) [reply examples/trivial.go]: See https://github.com/lightstep/lightstep-tracer-go/pull/191
